src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.external {
+  width: 328px;
+  background-color: #308dc0;
+  border: 1px solid #e95d49;
+  padding: 10px;
```

Comment:
You have border and padding styles the same for all three classes.
Don't repeat yourself, add common class with these styles.

Reply:
Really! DRY ))
Thanks!